Lib9c/TableData/ArenaSheet.cs

```diff
@@ -112,8 +112,22 @@ protected override void AddRow(int key, Row value)
         public Row GetRowByBlockIndex(long blockIndex) => OrderedList.First(e =>
             e.Round.Any(roundData => roundData.IsTheRoundOpened(blockIndex)));
 
-        public RoundData GetRoundByBlockIndex(long blockIndex) => OrderedList
-            .SelectMany(row => row.Round)
-            .First(e => e.IsTheRoundOpened(blockIndex));
+        public RoundData GetRoundByBlockIndex(long blockIndex)
+        {
+            var roundList = OrderedList.SelectMany(row => row.Round);
+
+            if (roundList == null || !roundList.Any())
+            {
+                throw new SheetRowNotFoundException(nameof(ArenaSheet), $"BlockIndex : {blockIndex}");
+            }
+
+            var round = roundList.First(e => e.IsTheRoundOpened(blockIndex));
+            if (round == null)
```

Comment:
i think we can 't handle this because `.First` will throw `InvalidOperationException`.